index.ts

```diff
@@ -1,283 +1,291 @@
-require("dotenv").config();
-var express = require("express");
-var http = require("http");
-var handlebars = require("express-handlebars");
-var popsicle = require('popsicle');
-var ClientOAuth2 = require('client-oauth2');
+import express = require("express");
+import http = require("http");
+import handlebars = require("express-handlebars");
+import popsicle = require('popsicle');
+import ClientOAuth2 = require('client-oauth2');
+import path = require("path");
+
+import morgan = require("morgan");
+import websocket = require("websocket");
+import bodyParser = require("body-parser");
+import dotenv = require("dotenv");
+import session = require("express-session");
+
+dotenv.config();
+
 
 var app = express();
-var server = http.Server(app);
+var server = new http.Server(app);
 
-var path = require("path");
 
-var morgan = require("morgan");
 app.use(morgan("common"));
 
-var session = require("express-session");
+
+
 app.use(session({
-  resave: false,
-  saveUninitialized: false,
-  secret: process.env.COOKIE_SECRET,
+    resave: false,
+    saveUninitialized: false,
+    secret: process.env.COOKIE_SECRET,
```

Comment:
This is one _massive_ commit but most of the diff comes purely from whitespace changes.
This makes reading the actual changes really hard, so please don't introduce unrelated whitespace changes - if you intend to do them, better do them in a separate style-only commit.

Reply:
Actually my intention was to introduce a consistent styling which included 4 space indentation and importing before writing any code. This commit was intended to split this styling change from all the other commits and was therefore introduced first, to make the following changes more comprehensible. But for my IDE to detect and rearrange the imports correctly, I hat to change the syntax to `import abc = require('xyz');`.
This should have been the only real code change in this commit and I associated this to the topic 'coding style'.

Reply:
> Actually my intention was to introduce a consistent styling which included 4 space indentation
But this is not clear from the commit and the diff shows massive changes - reviewing this is hard.

Reply:
I guess we can resolve this point by agreeing that style changes have nothing to do with the migration to typescript and therefore should not be part of this PR. I will see how I can elegantly undo these changes.
I would still keep the rearrangement of imports but with your [suggested style](https://github.com/BDSU/meeting-moderation-cards/pull/6#discussion_r541778653) (if possible)